debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -684,7 +641,14 @@ public LogMiningStrategy getLogMiningStrategy() {
      * @return whether log mining history is recorded
      */
     public Boolean isLogMiningHistoryRecorded() {
-        return getConfig().getBoolean(RECORD_LOG_MINING_HISTORY);
+        return logMiningHistoryRecorder != null;
```

Comment:
This is broken due to `resolveLogMiningHistoryRecorder`, is not it?

Reply:
Indeed, I had originally planned to have the recorder as `null` for the default but then determined it was cleaner to just create an empty implementation instead that did nothing to avoid unnecessary null checks.  I'll adjust.